### PR TITLE
Release 1.7.0

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

**Comment by praveen-garg** [comment]
We need to separate Core and Plugins change-logs.

**Comment by praveen-garg** [comment]
Also we should review README.rst especially 
https://github.com/foglamp/FogLAMP/blob/develop/README.rst#linux-distributions

**Comment by ashish-jabble** [comment]
@MarkRiddoch We should consider situation https://scaledb.atlassian.net/browse/FOGL-3071 JIRA for upgrade plugin users. This is only an issue of the plugins had been previously installed, old plugins should be manually removed form the system to use new released plugin version.

File: docs/91_version_history.rst [hunk 1]
```diff
@@ -25,6 +25,132 @@ Version History
 FogLAMP v1
 ==========
 
+v1.7.0
+-------
+
+Release Date: 2019-08-15
```
Comment (ashish-jabble) [inline docs/91_version_history.rst]: `2019-09-13`

File: docs/91_version_history.rst [hunk 2]
```diff
@@ -25,6 +25,132 @@ Version History
...
+       - The /foglamp/plugins REST API has been extended to allow plugins to be installed from an APT/RPM repository.
+       - Addition of support for hybrid plugins. A hybrid plugin is a JSON file that defines another plugin to load along with some default configuration for that plugin. This gives a means to create a new plugin by customising the configuration of an existing plugin. An example might be a plugin for a specific modbus device type that uses the generic modbus plugin and a predefined modbus map.
+       - The notification service has been improved to allow the re-trigger time of a notification to be defined by the user on a per notification basis.
+       - A new environment variable, FOGLAMP_PLUGIN_PATH has been added to allow plugins to be stored in multiple locations or locations outside of the usual FogLAMP installation directory.
+       - Added support for FOGLAMP_PLUGIN_PATH environment variable, that would be used for searching additional directory paths for plugins/filters to use with FogLAMP.
+       - FogLAMP packages for the Google Coral Edge TPU development board have been made available.
```
Comment (ashish-jabble) [inline docs/91_version_history.rst]: What about `odroid` based packages?

**Pull request merged**